includes/polling/ports.inc.php

```diff
@@ -201,6 +188,7 @@
                         $port_stats[$i] = $data[$i];
                     }
                     $port_stats = snmp_get_multi($device, $oids, '-OQUst', 'IF-MIB', null, $port_stats);
+                    $port_stats[$i]['polled'] = time();
```

Comment:
Before hand, we obtained the polled time for the ports way after all the snmpwalks when we actually only want to do it after ifEntry/ifXEntry. So we now set this either on a per port basis for selected ports polling or right after the if(X)Entry runs so we get as close to accurate time as possible.